check_stability.py

```diff
@@ -107,6 +106,48 @@ def __exit__(self, type, value, traceback):
         print("travis_fold:end:%s" % self.name, file=sys.stderr)
 
 
+class FilteredIO(StringIO):
```

Comment:
I think I follow you. I've pushed up a commit that makes the class inherit from "object" and implements the `__getattr__` method as a proxy. Let me know if this is what you had in mind.